src/ScaffoldInstallerPlugin.php

```diff
@@ -72,6 +72,130 @@ public static function getSubscribedEvents()
         ];
     }
 
+    /**
+     * Composer configuration advice: Use local copies of patch files.
+     *
+     * @return void
+     */
+    private function _checkComposerPatchesAreLocal()
+    {
+        $patchesInComposer = $this->extra['patches'] ?? false;
+        $patchesInExtraFile = $this->extra['patches-file'] ?? false;
```

Comment:
I think we should throw up a warning and discourage this as well since it won't get picked up by `composer validate` when patches are in a separate file

Reply:
Using a patches-file is discouraged on a separate check on this PR.